raster/raster-layer-file/README.md

```diff
@@ -1,21 +1,32 @@
-# Raster Layer File
+# Raster layer (file)
```

Comment:
I prefer the `(file)` in parentheses, and even better would be naming this sample `raster layer from file`.
However it now diverges from the original sample name. Do we want to go ahead with renaming anyway?

Reply:
after discussions about similar from the Android team, the renaming of samples will be another effort. So let's stick with this for now and then when the big name overhaul is done this can be updated then (along with our sample names in code).